ChiModules/KEigenvalueSolver/IterativeOperations/set_k_source.cc

```diff
@@ -105,11 +105,11 @@ void KEigenvalue::Solver::
             {
               for (size_t gprime=first_grp; gprime<=last_grp; ++gprime)
                 if (options.use_precursors)
-                  fission_g += xs->chi_g[g]*xs->nu_p_sigma_fg[gprime]*
-                               phi_prevp[gprime]/k_eff;
+                  fission_g += xs->chi[g] * xs->nu_prompt_sigma_f[gprime] *
```

Comment:
see my earlier comment regarding chi. Here, you call chi both the prompt chi and the steady-state chi

Reply:
Valid. See the response above. We will hit this via an issue.